projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,585 @@
+	"advancements.projectexpansion.infinite_fuel.description": "还是说是个十字架？管它呢，丢进熔炉里看看它的威力吧。",
+	"advancements.projectexpansion.collector": "更多的太阳能",
+	"advancements.projectexpansion.collector.description": "现在荒谬的事情开始了。",
+	"advancements.projectexpansion.compressed_collector": "收集器压缩之后变得没用了...",
+	"advancements.projectexpansion.compressed_collector.description": "...直到它被用来制作其他东西。",
+	"advancements.projectexpansion.final_collector": "终极收集器",
+	"advancements.projectexpansion.final_collector.description": "你真的需要这么多EMC吗？",
+	"advancements.projectexpansion.final_compressed_collector": "压缩终极收集器",
+	"advancements.projectexpansion.final_compressed_collector.description": "宇宙可能就此崩溃。",
+	"advancements.projectexpansion.relay": "迈向真正能量盆栽的第一步",
+	"advancements.projectexpansion.relay.description": "路途遥远。",
+	"advancements.projectexpansion.final_relay": "终极继电器",
+	"advancements.projectexpansion.final_relay.description": "真的，你需要这么多EMC吗？",
+	"advancements.projectexpansion.power_flower": "真正的能量盆栽",
+	"advancements.projectexpansion.power_flower.description": "现在你开始得心应手了。",
+	"advancements.projectexpansion.final_power_flower": "终极能量盆栽",
+	"advancements.projectexpansion.final_power_flower.description": "拜托，试着关掉游戏出去透透气吧。",
+	"advancements.projectexpansion.emc_link": "等等，你还能这么做？",
+	"advancements.projectexpansion.emc_link.description": "能量凝聚器不香了。",
+	"advancements.projectexpansion.final_emc_link": "终极EMC链接器",
+	"advancements.projectexpansion.final_emc_link.description": "帮我用EMC换点水来吧。",
+	"advancements.projectexpansion.final_star": "终极之星",
+	"advancements.projectexpansion.final_star.description": "大功告成，去外面走走吧。",
+	"advancements.projectexpansion.basic_alchemical_book": "炼金传送",
+	"advancements.projectexpansion.basic_alchemical_book.description": "获得一本基础炼金书。",
+	"advancements.projectexpansion.advanced_alchemical_book": "高级炼金书",
+	"advancements.projectexpansion.advanced_alchemical_book.description": "获得一本高级炼金书。",
+	"advancements.projectexpansion.master_alchemical_book": "大师炼金书",
+	"advancements.projectexpansion.master_alchemical_book.description": "获得一本大师炼金书。",
+	"advancements.projectexpansion.arcane_alchemical_book": "奥术炼金书",
+	"advancements.projectexpansion.arcane_alchemical_book.description": "获得一本奥术炼金书。",
+	"advancements.projectexpansion.knowledge_sharing_book": "分享即关爱！",
+	"advancements.projectexpansion.knowledge_sharing_book.description": "但我不在乎。",
+	"advancements.projectexpansion.compact_sun": "掌中太阳",
+	"advancements.projectexpansion.compact_sun.description": "别盯着它看太久……",
+	"advancements.projectexpansion.blinded_by_the_light": "被光芒所灼目",
+	"advancements.projectexpansion.blinded_by_the_light.description": "警告过你了。",
+	"block.projectexpansion.advanced_alchemical_chest.color_set": "颜色已设置为%s。",
+	"block.projectexpansion.advanced_alchemical_chest.invalid_item": "你必须持有%s才能设置颜色。",
+	"block.projectexpansion.advanced_alchemical_chest.color": "当前颜色：%s",
+	"block.projectexpansion.advanced_alchemical_chest.tooltip": "长得像箱子的炼金术之袋。用炼金术之袋Shift+右键箱子来设置颜色。",
```

```suggestion
	"block.projectexpansion.advanced_alchemical_chest.tooltip": "长得像箱子的炼金术之袋。用炼金术之袋Shift+右击箱子来设置颜色。",
```
**右键**不是一个完整的谓语，不能作为一个动作使用。
例如，`右键单击` 或 `右击` 、`右键长按` 或 `长按右键`都是可以的。但 `右键` 缺少语言成分。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。